submissions/Jemikalajah-Philip/javascript/homepage.js

```diff
@@ -0,0 +1,41 @@
+let imgDiv = document.getElementsByClassName("product--image");
```

Comment:
As discussed, the JS task should not depend on the existence of any divs already on your page. Clearly separate this, so it is clear at a glance what this task is all about.